master/buildbot/data/builds.py

```diff
@@ -95,8 +110,15 @@ def get(self, resultSpec, kwargs):
             builderid=kwargs.get('builderid'),
             buildrequestid=kwargs.get('buildrequestid'),
             complete=complete)
-        defer.returnValue(
-            [(yield self.db2data(dbdict)) for dbdict in builds])
+        # returns properties' list
+        filters = resultSpec.popFilter("property", "eq")
+        buildscol = []
+        for b in builds:
+            dbdict = yield self.db2data(b)
+            props = yield self.master.db.builds.getBuildProperties(b['id'])
+            self.injectProperties(dbdict, props, filters)
```

Comment:
``filters`` is the result of ``resultSpec.popFilter("property", "eq")``, which is a list of properties passed through url.

Meaning that requesting: 

+ **api/v2/builds?property=key1&property=key2** => allow to get the filtered builds' properties
+ **api/v2/builds?property=***  => allow to get the all builds' properties (wildcard)